_contentTemplates/common/get-started.md

```diff
@@ -13,59 +13,58 @@ The latest version of Telerik UI for Blazor is `{{site.uiForBlazorLatestVersion}
 #add-nuget-feed
 ### Add the Telerik NuGet Feed to Visual Studio
```

Comment:
The article relies heavily on using NuGet as a package source. Why do we include "#step-0---download-the-components" in the same article?

Reply:
This is extremely complicated. Downloading the components lets the Telerik Control Panel set up the nuget feed for people so they can usually skip parts of the tutorial. It also gives them the local demos and an option for a local feed in case they can't get to our online feed. That download section is also required by management to boost trial downloads. The whole tutorial is far too complex with far too many options and different config setups for it to be straightforward. For example, the trial experience from downloading the web installer is completely different from the setup for a person who has DCC and has had a commercial version for ages - the latter probably has the nuget feed setup already, for one, while the former sometimes does not even know what Telerik is.